src/components/dashboard/Dashboard.tsx

```
@@ -0,0 +1,212 @@
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from '@/components/ui/card'
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuLabel,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+} from '@/components/ui/dropdown-menu'
+import { Tabs, TabsList, TabsTrigger } from '@/components/ui/tabs'
+
+import dynamic from 'next/dynamic'
```

ESLint

[error] `next/dynamic` import should occur before import of `@/components/ui/badge`

(import/order)